Dockerfile

```diff
@@ -1,13 +1,11 @@
-FROM python:3.7-alpine
```

Comment:
why this change?

Reply:
@ableuler why are we changing the base image? Was this also a security vulnerability?

Reply:
Dependabot upgraded the cryptography package which doesn't play well with the alpine based image (we'd have to add a rust compiler). After switching to the debian based python image the package is pip-installable again. But I quickly wanted to try with the `-slim` base image and compare sizes, should have labeled it as WIP.

Reply:
https://github.com/pyca/cryptography/issues/5771

Reply:
The slim image is actually smaller than the resulting alpine image with the necessary installs.

Reply:
double win 🥇